content/en/docs/Getting Started/quickstart-kind.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Quickstart: Running Tekton on Kind"
+likTitle: "Quickstart: Running Tekton on Kind"
+weight: 1
+description: >
+  Set up and run your first Tekton task in 5 minutes with Kind
+---
+
+[Kind](https://kind.sigs.k8s.io/) is a tool for running Kubernetes locally using 
+Docker.
+
+## Prerequisites
+
+1.  [Install Docker](https://docs.docker.com/get-docker/)
+1.  [Install kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+1.  [Install kubectl](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+## Create your Kubernetes cluster
+
+Create a cluster named `tektoncd`:
+
+```bash
+kind create cluster --name tektoncd
+```
+
+The process takes a few seconds, you see the following output:
+
+<pre>
+Creating cluster "tektoncd" ...
+ ✓ Ensuring node image (kindest/node:v1.21.1) 🖼
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-tektoncd"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-tektoncd
+
+Have a nice day! 👋
+</pre>
+
+You can check that the cluster was successfully created with `kubectl`:
+
+```bash
+kubectl cluster-info --context kind-tektoncd
+```
+
+The output confirms that Kubernetes is running:
+
+<pre>
+Kubernetes control plane is running at https://127.0.0.1:39509
+CoreDNS is running at
+https://127.0.0.1:39509/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+</pre>
+
+## Install Tekton Pipelines
+
+1. To install the latest version of Tekton Pipelines, use `kubectl`:
+
+   ```bash
+   kubectl apply --context kind-tektoncd --filename \
+   https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+1. Monitor the installation:
+
+   ```bash
+   kubectl get pods --context kind-tektoncd --namespace tekton-pipelines --watch
+   ```
+
+When all components show `Running` under the `STATUS` column the installation
+is complete.
+
+Hit *Ctrl + C* to stop monitoring.
+
+## Create and run a basic Task
+
+A **Taks** defines a series of **steps** that run sequentially to perform some
+build work. Every Task runs as a pod on your Kubernetes cluster, with each step
+running in its own container.
+
+1. To create a task, open your favorite editor and create a file named
+   `hello-world.yaml` with the following content:
+
+	 ```yaml
+	 apiVersion: tekton.dev/v1beta1
+	 kind: Task
+	 metadata:
+	 	name: echo-hello-world
+	 spec:
+	 	steps:
+	 		- name: echo
+	 			image: alpine
+	 			command:
+	 				- echo
+	 			args:
+	 				- "Hello World"
```

Comment:
there's something odd about the indentation here - i think it's way to deep (2 spaces would probably be good, or 4 at most?) and it shows up as red in the preview 🤔 

![image](https://user-images.githubusercontent.com/432502/134740916-592f8f8c-444c-4b79-9a8d-eda2860ff873.png)

Reply:
I think these were tabs instead of spaces (?). This looked fine on my computer :). I made sure they were converted to spaces, I think it's fixed now.